airgun/views/contentview_new.py

```diff
@@ -0,0 +1,62 @@
+from wait_for import wait_for
+from widgetastic.widget import Checkbox
+from widgetastic.widget import Text
+from widgetastic.widget import TextInput
+from widgetastic_patternfly4.ouia import Button as PF4Button
+from widgetastic_patternfly4.ouia import ExpandableTable
+
+from airgun.views.common import BaseLoggedInView
+from airgun.views.common import SearchableViewMixinPF4
+
+
+class NewContentViewTableView(BaseLoggedInView, SearchableViewMixinPF4):
+    title = Text('.//h1[contains(@data-ouia-component-id, "OUIA-Generated-Text")]')
+    create_content_view = PF4Button('create-content-view')
+    table = ExpandableTable(
+        component_id='content-views-table',
+        column_widgets={
+            'Name': Text('./a'),
+            'Last Published': Text('./a'),
+            'Last task': Text('.//a'),
+            'Latest version': Text('.//a'),
+        },
+    )
+
+    @property
+    def is_displayed(self):
+        return self.browser.wait_for_element(self.create_content_view, exception=False) is not None
+
+
+class NewContentViewCreateView(BaseLoggedInView):
+    title = Text("//h1[contains(., 'Create content view') or contains(@id, 'pf-modal-part-1')]")
+    name = TextInput(id='name')
+    label = TextInput(id='label')
+    description = TextInput(id='description')
+    # cv and ccv are tiles
+    component = Text('//div[contains(@id, "component")]')
+    composite = Text('//div[contains(@id, "composite")]')
+    solve_dependencies = Checkbox(id='dependencies')
+    import_only = Checkbox(id='importOnly')
+    auto_publish = Checkbox(id='auto_publish')
+    submit = PF4Button('create-content-view-form-submit')
+    cancel = PF4Button('create-content-view-form-cancel')
+
+    @property
+    def is_displayed(self):
+        return self.browser.wait_for_element(self.title, exception=False) is not None
+
+    def after_fill(self, value):
+        """Ensure 'Create content view' button is enabled after filling out the required fields"""
+        wait_for(
+            lambda: self.submit.is_displayed is True, timeout=300, delay=1, logger=self.logger
```

Comment:
`is True` is redundant here. wait_for's fail_condition is going to be `False`, so its looping on `(self.submit.is_displayed is True) == False`, which is the same as just looping on `self.submit.is_displayed == False`, since `is_displayed` returns a boolean.

That aside, there's no need for all this. Just call `self.submit.wait_displayed()`